### DISCUSSION TITLE
Feature request: simulation of dueling coasters

### DISCUSSION THREAD

**Discussion by galfisk** [OpenRCT2/OpenRCT2#15776] [Feature requests]
When building a dueling coaster, it would be useful if one ghost train started from each platform, so that one could see how they would interplay. It could be enabled by ticking the Synchronize adjacent stations box. It'd also be useful if, on real testing or opening, the actual trains started out spread across the stations. Furthermore, it'd be useful if, when a simulated train reached the end of a track, only that train disappeared and returned to the station, instead of all trains at once, especially when building a long coaster.

**Comment by spacek531** [comment]
Is this for mobius coasters or separate coasters?

**Comment by galfisk** [comment]
Mobius. It already works for separate coasters.

**Comment by MASHzj** [comment]
This could be resolved if [#17842](https://github.com/OpenRCT2/OpenRCT2/discussions/17842) is implemented.
